src/main/java/io/zeebe/http/ZeebeHttpWorkerConfig.java

```diff
@@ -26,6 +26,9 @@
   @Value( "${ENV_VARS_M2M_AUDIENCE:#{null}}")
   private String environmentVariablesM2mAudience;  
 
+  @Value( "${LOCAL_ENV_PREFIX:WORKFLOW_ENV_}")

```

Comment:
What do you think about changing the prefix to `ZEEBE_`? It may be more clear that these variables should be accessed in the Zeebe context.

```suggestion
  @Value( "${LOCAL_ENV_PREFIX:ZEEBE_}")
```

Reply:
Maybe something like `ZEEBE_ENV_` would be better to avoid including zeebe's sensitive env variables like `ZEEBE_CLIENT_SECRET`